### PR TITLE
🌐 Add Ukrainian translation for `docs/uk/docs/tutorial/extra-data-types.md`

### PR DESCRIPTION
**Pull request by ArtemKhymenko**
Context:
Ukrainian translation for docs/tutorial pages
Changes:
Created Ukrainian version of docs/tutorial/extra-data-types page

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 0f69e164c0b171a79648dd7e70e048c936a33a55 at: https://7e1c92a5.fastapitiangolo.pages.dev

**Review by rostik1410 — APPROVED** [review]

**Review by mezgoodle — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit cb78037710af66e8bd70e451b53cdc6d63e1661b at: https://75ee0bd9.fastapitiangolo.pages.dev

**Comment by ArtemKhymenko** [comment]
@tiangolo @rostik1410 @mezgoodle @rgilton Hey! I've added docs/uk/docs/tutorial/encoder.md here, can you, please, review it. Thank's!
* next ones will commit as another branches and new PR.

File: docs/uk/docs/tutorial/encoder.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Існують випадки, коли вам може знадобитися перетворити тип даних (наприклад, модель Pydantic) в щось сумісне з JSON (наприклад, `dict`, `list`, і т. д.).
+
+Наприклад, якщо вам потрібно зберегти це в базі даних.
+
+Для цього, **FastAPI** надає `jsonable_encoder()` функцію.
+
+## Використання `jsonable_encoder`
+
+Давайте уявимо, що у вас є база даних `fake_db`, яка приймає лише дані, сумісні з JSON.
```
Comment (rostik1410) [inline docs/uk/docs/tutorial/encoder.md:11]:
```suggestion
Уявімо, що у вас є база даних `fake_db`, яка приймає лише дані, сумісні з JSON.
```
as an alternative variant, your variant also good

File: docs/uk/docs/tutorial/encoder.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Наприклад, вона не приймає об'єкти типу `datetime`, оскільки вони не сумісні з JSON.
+
+Отже, об'єкт типу `datetime` потрібно перетворити в рядок `str`, який містить дані в <a href="https://en.wikipedia.org/wiki/ISO_8601" class="external-link" target="_blank">ISO формат</a>.
```
Comment (rostik1410) [inline docs/uk/docs/tutorial/encoder.md:15]:
```suggestion
Отже, об'єкт типу `datetime` потрібно перетворити в рядок `str`, який містить дані в <a href="https://en.wikipedia.org/wiki/ISO_8601" class="external-link" target="_blank">ISO форматі</a>.
```

**Review by rostik1410 — COMMENTED** [review]
Looks good, only one typo I noticed (look at comments). Also, the repo owner usually asks to split PRs by files (1 file - 1 PR)

**Review by mezgoodle — APPROVED** [review]

File: docs/uk/docs/tutorial/encoder.md (hunk 2 above)
Comment (rostik1410) [reply docs/uk/docs/tutorial/encoder.md:15]: @ArtemKhymenko could you pls fix this typo?

**Comment by tiangolo** [comment]
📝 Docs preview for commit 9ad50c14673826958f011a0e107ff3d24aafc000 at: https://3b71cc98.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit cb2dd852badbd4fa6e8433cac510677648ed3a94 at: https://b980410f.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
Thank you @ArtemKhymenko! 🎉 

And thanks for the reviews @rostik1410 and @mezgoodle 🙇

**Comment by alekna** [comment]
Ukrainian country code is `ua`, not `uk`.

**Comment by rostik1410** [comment]
@alekna country code yes, but language code no)

**Pull request merged**